module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -1126,21 +1126,35 @@ public function deletelistAction()
     }
 
     /**
-     * Get a record driver object corresponding to an array returned by an ILS
-     * driver's getMyHolds / getMyTransactions method.
+     * Get record driver objects corresponding to an of record arrays returned by an
```

Comment:
I think there's a missing word here -- should this be "array of record arrays?"

Reply:
Fixed.